__tests__/app.test.js

```diff
@@ -114,29 +114,41 @@ describe("* GET/api/articles/:article_id", () => {
   });
 
   test("responds with a single matching article", () => {
-    return request(app).get("/api/articles/3").expect(200)
-    .then(({ body }) => {
-      expect(body).toEqual(
-        {
-          article_id: 3,
-          title: 'Eight pug gifs that remind me of mitch',
-            topic: 'mitch',
-            author: 'icellusedkars',
-            body: 'some gifs',
-            created_at: '2020-11-03T09:12:00.000Z',
-            votes: 0
-        }
-      )
-    });
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
+        );
+      });
```

Comment:
I think there may have been some confusion regarding Poonam's last comments...
You were totally correct to assert for specific values as we are requesting a specific article by its ID.
What Poonam was suggesting was to wrap that in a `expect.objectContaining` just to allow for further properties to be present in future